tools/st2rc.sh

```diff
@@ -0,0 +1,15 @@
+echo -n "Please enter your StackStorm username: "
+read ST2_USERNAME
+export ST2_USERNAME=$ST2_USERNAME
```

Comment:
I believe CLI doesn't support st2_username and st2_password environment variable so we probably shouldn't export it (also for the security reasons).

(re-posting comment from #1410)